### PR TITLE
Moves RequestCodes to ApplicationConstants

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Just a minor refactor. The main purpose for doing this is to reduce the size of the FormEntryActivity. 

#### What has been done to verify that this works as intended?
Pure code refactor, does not alter the behavior in any way

#### Why is this the best possible solution? Were any other approaches considered?
Since RequestCode for the various widgets is accessed in multiple classes, not just in FormEntryActivty. So moving them to application constants seemed like a good option. 

#### Are there any risks to merging this code? If so, what are they?
No

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java [hunk 1]
```diff
@@ -152,29 +153,6 @@
     private static final boolean EVALUATE_CONSTRAINTS = true;
     private static final boolean DO_NOT_EVALUATE_CONSTRAINTS = false;
 
-    // Request codes for returning data from specified intent.
-    public static final int IMAGE_CAPTURE = 1;
```
Comment (heyjamesknight) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java:157]: Yessssssssssssss, thanks @shobhitagarwal1612

**Review by heyjamesknight — APPROVED** [review]
Looks great, no moving changes. Nice to have those Constants out of the FEA, thanks @shobhitagarwal1612 !

**Comment by heyjamesknight** [comment]
My only comment: I do wonder if its more clear to leave the Constants qualified with `RequestCodes`, as it helps make it clear that is where all of the codes belong.

I think the commit is fine as-is, just wanted to bring this up for discussion.

**Review by lognaturel — APPROVED** [review]

**Comment by lognaturel** [comment]
Thanks, @shobhitagarwal1612! Good point from @jknightco and I filed #1537 to handle it separately.

**Pull request merged**